mewarpx/mewarpx/utils_store/profileparser.py

```diff
@@ -0,0 +1,160 @@
+from os import write
+import sys
+import re
+import json
+
+
+class FullProfile(object):
+    """
+    Class for parsing, storing and writing the full profile data
+    """
+    def __init__(self, lines, stdout_path, write_dir):
```

Comment:
Maybe making `write_dir` an optional argument (with default `None`) would be better, then just checking below if it is `None` and if so set it to be the same as `self.stdout_path`.